compiler/passes/ResolveScope.cpp

```diff
@@ -624,13 +624,21 @@ Symbol* ResolveScope::lookupForImport(Expr* expr) const {
           // if we're not in the root module scope or using relative import,
           // this is an improper match
           badCloserModule = mod;
+          if (isUse) { // TODO: remove this to disable relative use
+            retval = sym;
+            break;
+          }
         } else {
           // if we are in the root module scope, then it is a proper match.
           retval = sym;
           break;
         }
       } else if (sym != NULL) {
         // found something other than a module
+        if (isUse && isTypeSymbol(sym) && isEnumType(sym->type)) {
+          retval = sym;
+          break;
+        }
         USR_FATAL(expr, "import must name a module ('%s' not a module)",
```

Comment:
This error message should be adjusted to respond to whether we're resolving a use or import statement